Dockerfile.template

```diff
@@ -1,39 +1,33 @@
 ARG ARCH=%%BALENA_ARCH%%
-
-# Used by livepush to support multi arch images in older
-# balenaOS with buggy platform support
-# see https://github.com/balena-os/balena-engine/issues/269
-ARG PREFIX=library
 ARG FATRW_VERSION=0.2.9
 
 ###################################################
 # Build the supervisor dependencies
 ###################################################
-FROM balenalib/${ARCH}-alpine-node:16-run as build-base
+FROM alpine:3.16 as build-base
 
 ARG ARCH
-ARG PREFIX
 ARG FATRW_VERSION
 ARG FATRW_ARCHIVE="fatrw-${ARCH}.tar.gz"
 ARG FATRW_LOCATION="https://github.com/balena-os/fatrw/releases/download/v${FATRW_VERSION}/${FATRW_ARCHIVE}"
-# Sanity check to prevent a prefix for a non-official docker image being
-# inserted. Only 'library' and 'arm32v6' are allowed right now
-RUN for allowed in "library" "arm32v6"; do [ "${PREFIX}" = "${allowed}" ] && break; done
 
 WORKDIR /usr/src/app
 
-RUN apk add --no-cache \
+RUN apk add --update --no-cache \
 	g++ \
 	make \
 	python3 \
-	libgcc \
+	curl \
+	'nodejs<18' \
+	'npm<9' \
```

Comment:
Should we define the Node / NPM version at the top of the Dockerfile as an ARG for easy updating? I see it's referenced in multiple places.

Reply:
Otherwise, LGTM. Good work!

Reply:
👍

Reply:
Good idea, I'll make the change